test/signingserver_test.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+
+import signingserver
+
+
+def test_distinguished_name():
+    assert (
```

Comment:
codestyle: readability might be improved by escaping linebreaks, rather than using parentheses, wdyt?